utils/src/main/kotlin/LicenseFilenamePatterns.kt

```diff
@@ -47,12 +47,18 @@ object LicenseFilenamePatterns {
         "readme*"
     ).generateCapitalizationVariants()
 
+    /**
+     * A list of globs that match all kind of license file names, equaling the union of [LICENSE_FILENAMES] and
+     * [ROOT_LICENSE_FILENAMES].
+     */
+    val ALL_FILENAMES = LICENSE_FILENAMES + ROOT_LICENSE_FILENAMES
```

Comment:
`ALL_LICENSE_FILENAMES` would be a better name, `ALL_FILENAMES` sounds like it would really match everything.

Reply:
Done.